botocore/waiter.py

```diff
@@ -324,12 +327,17 @@ def wait(self, **kwargs):
                 if 'Error' in response:
```

Comment:
I don't think it makes sense to treat any response with an `Error` key as an error In this case we should be treating it as a success response, since from an SDK/HTTP perspective the API call was successful it just happens to have a member named `Error`.